NextcloudTalk/SettingsTableViewController.swift

```diff
@@ -0,0 +1,696 @@
+        }
+        return true
+    }
+
+    // MARK: Configuration
+
+    func presentVideoResoultionsSelector() {
+        let videoConfIndexPath = self.getIndexPathForConfigurationOption(option: ConfigurationSectionOption.kConfigurationSectionOptionVideo)
+        let videoResolutions = NCSettingsController.sharedInstance().videoSettingsModel.availableVideoResolutions()
+        let storedResolution = NCSettingsController.sharedInstance().videoSettingsModel.currentVideoResolutionSettingFromStore()
+
+        let optionsActionSheet = UIAlertController(title: NSLocalizedString("Video quality", comment: ""), message: nil, preferredStyle: .actionSheet)
+
+        for resolution in videoResolutions {
+            let readableResolution = NCSettingsController.sharedInstance().videoSettingsModel.readableResolution(resolution)
+            let isStoredResolution = resolution == storedResolution
+            let action = UIAlertAction(title: readableResolution, style: .default) { _ in
+                NCSettingsController.sharedInstance().videoSettingsModel.storeVideoResolutionSetting(resolution)
+                self.tableView.beginUpdates()
+
+                self.tableView.reloadRows(at: [videoConfIndexPath], with: .none)
+
+                self.tableView.endUpdates()
+            }
+
+            if isStoredResolution {
+                action.setValue(UIImage(named: "checkmark")?.withRenderingMode(_:.alwaysOriginal), forKey: "image")
+            }
+            optionsActionSheet.addAction(action)
+        }
+
+        optionsActionSheet.addAction(UIAlertAction(title: NSLocalizedString("Cancel", comment: ""), style: .cancel, handler: nil))
+
+        // Presentation on iPads
+        optionsActionSheet.popoverPresentationController?.sourceView = self.tableView
+        optionsActionSheet.popoverPresentationController?.sourceRect = self.tableView.rectForRow(at: videoConfIndexPath)
+
+        self.present(optionsActionSheet, animated: true, completion: nil)
+    }
+
+    func presentBrowserSelector() {
+        let browserConfIndexPath = self.getIndexPathForConfigurationOption(option: ConfigurationSectionOption.kConfigurationSectionOptionBrowser)
+        let supportedBrowsers = NCSettingsController.sharedInstance().supportedBrowsers!
+        let defaultBrowser = NCUserDefaults.defaultBrowser()
+
+        let optionsActionSheet = UIAlertController(title: NSLocalizedString("Open links in", comment: ""), message: nil, preferredStyle: .actionSheet)
+
+        for browser in supportedBrowsers {
+            var isDefaultBrowser = false
+            if let browser = browser as? String {
+                isDefaultBrowser = browser == defaultBrowser
+                let action = UIAlertAction(title: browser, style: .default) { _ in
+                    NCUserDefaults.setDefaultBrowser(browser)
+                    self.tableView.reloadData()
+                }
+                if isDefaultBrowser {
```

Comment:
This is a total nitpick, but I don't think the variable adds a lot of information compared to just `if browser == defaultBrowser {` :)